### PR TITLE
Fix/minor style issues

### PR DESCRIPTION
**Pull request by 0x3bfc**
- [x] solidity styling 
- [x] missing license 
- [x] integrating Solium

### CODE REVIEW

**Comment by sept-en** [comment]
LGTM! It's good to have style/linter here.

However I'd prefer if we set the `max-len` in the config so not all statements (e.g. `require`) are bloated up into several lines if they are quite small.

Also I don't think this is the scope of #16 PR. I'd prefer to wait until #16 is merged and point the current PR to the `master`.

File: eth-custodian/.soliumrc.json
Comment (sept-en) [inline eth-custodian/.soliumrc.json:10]: Let's set this to at least 120.

**Comment by 0x3bfc** [comment]
> However I'd prefer if we set the `max-len` in the config so not all statements (e.g. `require`) are bloated up into several lines if they are quite small.

The max length so far is `95` in the `.soliumrc.json` ... For the `require` I just wanted to make it simple to read and differentiate between the condition and the error message. 

> Also I don't think this is the scope of #16 PR. I'd prefer to wait until #16 is merged and point the current PR to the `master`.

No problem. Feel free to rebase the PR whenever the PR #16 is merged.

File: eth-custodian/.soliumrc.json
Comment (0x3bfc) [reply eth-custodian/.soliumrc.json:10]: Ok

File: eth-custodian/contracts/ProofKeeper.sol
Comment (sept-en) [inline eth-custodian/contracts/ProofKeeper.sol:83]: `           keccak256(fullOutcomeProof.outcome_proof.outcome_with_id.outcome.executor_id)
            == keccak256(nearProofProducerAccount_),`

If we leave the equal sign it on the previous line that has a negative effect on the readability and will look like this is just another argument to the function.

Comment (0x3bfc) [reply eth-custodian/contracts/ProofKeeper.sol:83]: I agree ... I only did that because the linter had a warning by adding the == in the next line.

A good way is to avoid this by assigning the two parties to a memory parameters.
```javascript
param1 = keccak256(....x1);
param2 = keccak256(....x2);
require(param1 == param2, error_msg);
```

File: eth-custodian/contracts/test/NearProverMock.sol
Comment (sept-en) [inline eth-custodian/contracts/test/NearProverMock.sol:27]: Please check `returns(bool` indentation and put `{` on the new line

File: eth-custodian/contracts/test/NearProverMock.sol
Comment (sept-en) [inline eth-custodian/contracts/test/NearProverMock.sol:14]: Please check `returns(bool` indentation and put `{` on the new line

**Comment by sept-en** [comment]
Btw, you need to run `dist.sh` script to update the contract dist & res files. I think now the values were manually changed there.

**Comment by 0x3bfc** [comment]
> Btw, you need to run `dist.sh` script to update the contract dist & res files. I think now the values were manually changed there.

Sure  👍 
I was thinking about releasing these contracts as an ABI and anyone can use them by just installing the pypi/npm package.

**Comment by sept-en** [comment]
>  I was thinking about releasing these contracts as an ABI and anyone can use them by just installing the pypi/npm package.

That would be good! However, this makes sense as soon as we release some stable version and mark it with the appropriate tag.

**Review by sept-en — APPROVED** [review]

**Pull request merged**
